src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java

```diff
@@ -50,18 +50,38 @@ public void bindCommands() {
 					new SetEnableableModifier(modifier, true)));
 		RobotMap.Component.driverXbox.lb.whenReleased(new SetEnableableModifier(modifier, true));
 		Command normalDrive = new ChassisMove(RobotMap.Component.chassis, this);
+		// Down
+		RobotMap.Component.driverXbox.dPad.down.whenPressed(new ChassisTurnAbsolute(RobotMap.Component.chassis, 0,
+			RobotMap.Component.navx, RobotMap.Component.chassisDriveMC));
+		RobotMap.Component.driverXbox.dPad.down.whenReleased(normalDrive);
+		// Down-Right
+		RobotMap.Component.driverXbox.dPad.downRight.whenPressed(new ChassisTurnAbsolute(RobotMap.Component.chassis, 45,
+			RobotMap.Component.navx, RobotMap.Component.chassisDriveMC));
+		RobotMap.Component.driverXbox.dPad.downRight.whenReleased(normalDrive);
+		// Right
+		RobotMap.Component.driverXbox.dPad.right.whenPressed(new ChassisTurnAbsolute(RobotMap.Component.chassis, 90,
+			RobotMap.Component.navx, RobotMap.Component.chassisDriveMC));
+		RobotMap.Component.driverXbox.dPad.right.whenReleased(normalDrive);
+		// Up-Right
+		RobotMap.Component.driverXbox.dPad.upRight.whenPressed(new ChassisTurnAbsolute(RobotMap.Component.chassis, 135,
+			RobotMap.Component.navx, RobotMap.Component.chassisDriveMC));
+		RobotMap.Component.driverXbox.dPad.upRight.whenReleased(normalDrive);
+		// Up
 		RobotMap.Component.driverXbox.dPad.up.whenPressed(new ChassisTurnAbsolute(RobotMap.Component.chassis, 180,
 			RobotMap.Component.navx, RobotMap.Component.chassisDriveMC));
 		RobotMap.Component.driverXbox.dPad.up.whenReleased(normalDrive);
-		RobotMap.Component.driverXbox.dPad.down.whenPressed(new ChassisTurnAbsolute(RobotMap.Component.chassis, 0,
+		// Up-Left
+		RobotMap.Component.driverXbox.dPad.upLeft.whenPressed(new ChassisTurnAbsolute(RobotMap.Component.chassis, 225,
 			RobotMap.Component.navx, RobotMap.Component.chassisDriveMC));
-		RobotMap.Component.driverXbox.dPad.down.whenReleased(normalDrive);
+		RobotMap.Component.driverXbox.dPad.upLeft.whenReleased(normalDrive);
+		// Left
 		RobotMap.Component.driverXbox.dPad.left.whenPressed(new ChassisTurnAbsolute(RobotMap.Component.chassis, 270,
 			RobotMap.Component.navx, RobotMap.Component.chassisDriveMC));
 		RobotMap.Component.driverXbox.dPad.left.whenReleased(normalDrive);
-		RobotMap.Component.driverXbox.dPad.right.whenPressed(new ChassisTurnAbsolute(RobotMap.Component.chassis, 90,
+		// Down-Left
+		RobotMap.Component.driverXbox.dPad.downLeft.whenPressed(new ChassisTurnAbsolute(RobotMap.Component.chassis, 315,
 			RobotMap.Component.navx, RobotMap.Component.chassisDriveMC));
-		RobotMap.Component.driverXbox.dPad.right.whenReleased(normalDrive);
+		RobotMap.Component.driverXbox.dPad.downLeft.whenReleased(normalDrive);
```

Comment:
@billwpierce  I agree with you on that. Given the limited amount of time, we can return to this after build season is over if we want to clean it up. For now, if it works, it can be good enough.